model/user/db.go

```diff
@@ -189,11 +189,15 @@ func FindByRole(role string) ([]DBUser, error) {
 	return res, errors.Wrapf(err, "error finding users with role '%s'", role)
 }
 
-func FindByRoles(roles []string) ([]DBUser, error) {
+// FindUsersByRoles returns only real users (i.e. omits API only users).
+func FindUsersByRoles(roles []string) ([]DBUser, error) {
```

Comment:
The service-user-only queries include the word service in them to make it obvious that they only apply to the bots. Can we call this `FindHumanUsersByRoles` (or something to that effect) to distinguish it from queries that also include service users?

Reply:
Yes perfect I couldn't think of a good name for this